components/developer/jenkins/files/jenkins-standalone.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version='1.0'?>
+<!--
+//
+//
+// This file and its contents are supplied under the terms of the
+// Common Development and Distribution License ("CDDL"), version 1.0.
+// You may only use this file in accordance with the terms of version
+// 1.0 of the CDDL.
+//
+// A full copy of the text of the CDDL should have accompanied this
+// source.  A copy of the CDDL is also available via the Internet at
+// http://www.illumos.org/license/CDDL.
+//
+//
+// Copyright 2016-2017 by Jim Klimov, fixes 2017 added by Adam Stevko
```

Comment:
Please, make it sane. Either remove @xen0l if he is not against it or put him on separate line.